catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/impl/CreateRequestImpl.java

```diff
@@ -83,7 +86,7 @@ public CreateRequestImpl(
    * @see ddf.catalog.operation.CreateRequest#getMetacards()
    */
   @Override
-  public List<Metacard> getMetacards() {
+  public Set<Metacard> getMetacards() {
```

Comment:
❓ Were there any tests for this? Should new ones be added? I see that we have tests with their own implementations of the interface. Was updating those sufficient?